### ISSUE TITLE
Facebook APIs v2.4 and SDK 5 php

### ISSUE THREAD

**Issue by Nimesh-Soni** [hybridauth/hybridauth#524]
hello there,

facebook Announcing Marketing APIs v2.4 and sdk5 recently.
Does it compatible for that?
Because when i create facebook app it will automatically created by v2.4 and only response "id and Full name".
Does it work for that ?

**Comment by askaidau** [comment]
Names, here's a quick stop-gap fix, change line 163 of Hybrid/Providers/Facebook.php:
```
try{ 
    $data = $this->api->api('/me?fields=birthday,email,name,link,gender,hometown');
}
```

**Comment by lilbran** [comment]
@askaidau Thank you so much! You've solved my issue! Is this only because of the update from v2.3 to v2.4? I just downloaded the hybridauth library today, surprised this hasn't been included.

**Comment by askaidau** [comment]
@lilbran Glad to help. Yes you are right, it was a breaking change introduced in v2.4 (fewer default fields for faster performance) https://developers.facebook.com/blog/post/2015/07/08/graph-api-v2.4/

**Comment by AdwinTrave** [comment]
This should be fixed by #525
